docs/security-policy.md

```diff
@@ -139,4 +139,4 @@ Limitations of SecurityPolicy CR:
    given NSX-T does not support 'In' in NSGroup condition, so we use a workaround to support 'In' with limited counts.
 7. Max IP elements in one security policy: 4000
 8. Priority range of SecurityPolicy CR is [0, 1000].
-9. Do not support named port.
```

Comment:
t's better to add a named port in this example above since we want to cover usage of SecurityPolicy as much as possible in this one, including matchLables, matchExpressions, namespaceSelector, vmSelector,podSelector and named port etc. It will be a good reference for users.